src/googleclouddebugger/module_explorer.py

```diff
@@ -62,7 +62,6 @@ def GetCodeObjectAtLine(module, line):
         prev_line = max(prev_line, co_line_number)
       elif co_line_number > line:
         next_line = min(next_line, co_line_number)
```

Comment:
The break is removed because lines can now go backwards? Maybe add a comment here saying the search continues since it's possible for lines to go backwards and it may yet be found.

Reply:
Added a comment